### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by srronaksingh**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by srronaksingh — COMMENTED** [review]
You won't be able to approve or request changes to your own pull request.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ _Collaborate and work together on GitHub._
   Do not use quotes on the <details> tag attributes.
 -->
```
Comment (srronaksingh) [inline README.md:20]: '''sugestion
nothing
'''

File: README.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Next, wait about 20 seconds for actions to run, then refresh this course's READM
   Historic note: this step combines the commend, approve, and needs changes steps from the previous version.
 -->
```
Comment (RonakSingh-MLSA) [inline README.md:82]: '''suggestion
nothing
'''

**Review by RonakSingh-MLSA — COMMENTED** [review]
nothing

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (RonakSingh-MLSA) [inline index.html:49]: '''suggestion
    nothing
'''

**Review by RonakSingh-MLSA — COMMENTED** [review]
'''suggestion
nothing
'''

**Comment by srronaksingh** [comment]
.

**Pull request merged**